pyartcd/pyartcd/git.py

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
empty line

Reply:
Nice spot. I think a linter like flake8 needs to be added to the test. For now I would like to hold on that because it will complain a lot. So better to do that in a separate PR.

Reply:
Added flake8 test.